### PR TITLE
do_restore_write_mbr without fixing_gpt flag

### PR DESCRIPTION
**Pull request by cagnulein**
as referenced here https://forums.urbackup.org/t/writing-gpt-backup-header-failed-error-on-restore/14451
I guess the code in the client_restore.cpp it was outdate about the fix mbr parameter.

### CODE REVIEW

File: urbackupserver/www/translations/urbackup.webinterface/it_IT.po [hunk 1]
```diff
@@ -808,7 +808,7 @@ msgid "tDo image backups over internet"
 msgstr "Esegui backup immagine attraverso Internet"
 
 msgid "tDo full file backups over internet"
-msgstr "Esegui backup file atraverso Internet"
+msgstr "Esegui backup file attraverso Internet"
```
Comment (uroni) [inline urbackupserver/www/translations/urbackup.webinterface/it_IT.po:811]: Changes here will be overwritten once the files are updated from Transifex

**Comment by uroni** [comment]
The parameter is false in non-restore CD cases because there is no guarantee that `sgdisk` is available and the "fix" liberally relies on this.

Perhaps we can make the fix a parameter that is false by default?

File: urbackupserver/www/translations/urbackup.webinterface/it_IT.po (hunk 1 above)
Comment (cagnulein) [reply urbackupserver/www/translations/urbackup.webinterface/it_IT.po:811]: hah ok, so where should I do this change?

**Comment by cagnulein** [comment]
> The parameter is false in non-restore CD cases because there is no guarantee that `sgdisk` is available and the "fix" liberally relies on this.
> 
> Perhaps we can make the fix a parameter that is false by default?

or maybe we can test the presence of sgdisk and set the parameter based on this and also maybe outputting something about this on the terminal?

**Pull request merged**